doc/ref_arch/kubernetes/chapters/chapter06.md

```diff
@@ -58,15 +58,15 @@ The following are core principles to consider when securing cloud native applica
 Ensure kubernetes nodes are secure, hardened and configured correctly following well known security framework e.g. CIS benchmark, etc. Administrative access to kubernetes nodes should be restricted while operational activities including debugging, troubleshoting, and other tasks should be handled without direct access to the nodes.
 
 ##  6.4 Authentication & Authorisation
-All connections to a Kubernetes cluster must be via a secure channel. The following security authentication mechanism must be adopted;
+All connections to a Kubernetes cluster must be via a secure channel. The following security authentication mechanism must be adopted:
 
  - User roles and access levels must be configured to provide segregation of duties (RBAC)
  - Multi-factor authentication is mandated for all administrative access
  - Service and session authentication mechanisms must be token-based or certificate-based
  - Access control should be integrated with existing identity management platforms e.g SAML, AD, etc.
 
 ##  6.5 Use Namespaces to Establish Security Boundaries
-Namespaces in Kubernetes is the first level of isolation between components. It is easier to apply security controls (Network Policies, Pod policies, etc) to diffferent types of workloads when deployed in seperate namespaaces. 
+Namespaces in Kubernetes is the first level of isolation between components. It is easier to apply security controls (Network Policies, Pod policies, etc) to diffferent types of workloads when deployed in seperate namespaces. 
```

Comment:
One more typo: seperate ==> separate, also in next line.

Reply:
fixed in : https://github.com/cntt-n/CNTT/pull/1375/commits/6d84c8b8b12b0346158751f8c45f66ea1e3d89d1